packages/core/src/designs/design-processor.ts

```diff
@@ -57,7 +57,7 @@ export class DesignProcessor {
     let designInnerHtml = '';
 
     for (const key in design) {
-      if (design.hasOwnProperty(key)) {
+      if (design.hasOwnProperty(key) && (design as any)[key]) {
```

Comment:
A general direction we want to take is try to be as explicit as possible. is `design[key]` supposed to be always string? If so we should do `design[key] !== ''`

Reply:
That's a good question. Theoretically it should always be a string. However it seems that it could also happen to be `undefined` and in this case I would prefer to ignore the key. Because if it keep going, the `designInnerHTML` will contain invalid css code and result in wrong rendering.

Reply:
We could do:

```
if (design.hasOwnProperty(key) && typeof (design as any)[key] === 'string' && (design as any)[key] !== '')
```

Reply:
@bigopon 
Fix: https://github.com/platform5/ux/commit/c1ad6c64eb73d96490021b8dfeec8acc3deb571d